pandas/conftest.py

```diff
@@ -78,6 +82,13 @@ def pytest_runtest_setup(item):
         pytest.skip("skipping high memory test since --run-high-memory was not set")
 
 
+def pytest_collection_modifyitems(config, items):
```

Comment:
does this show up in a lot of places?

Reply:
Yes.  I was putting `pytestmark = ...` in dozens of files and barely making a dent before putting it here instead